src/main/webapp/app/core/about-us/about-us.component.html

```diff
@@ -0,0 +1,58 @@
+<div class="card-title">
+    <h2 class="mb-3 mt-3 font-weight-bold">{{ 'aboutUsOverview.title' | translate }}</h2>
+</div>
+<div class="mb-5">
+    <h4 class="mb-3 font-weight-bold">{{ 'aboutUsOverview.feature' | translate }}</h4>
+    <h6>{{ 'aboutUsOverview.exercises.title' | translate }}</h6>
+    <ul>
+        <li>{{ 'aboutUsOverview.exercises.detail.programmingExercises' | translate }}</li>
+        <li>{{ 'aboutUsOverview.exercises.detail.quiz' | translate }}</li>
+        <li>{{ 'aboutUsOverview.exercises.detail.modelingExercises' | translate }}</li>
+        <li>{{ 'aboutUsOverview.exercises.detail.textExercises' | translate }}</li>
+        <li>{{ 'aboutUsOverview.exercises.detail.uploadExercises' | translate }}</li>
+    </ul>
+</div>
+
+<div class="row ml-4 mr-4 mb-4" *ngIf="data && data.projectManagers">
+    <div class="col-md-6 grid-margin" *ngFor="let pm of data.projectManagers">
+        <div class="card-body">
+            <div class="row ml-1">
+                <div class="col-md-4"><img class="img-fluid w-100 rounded" [src]="SERVER_API_URL + pm.photoDirectory" /></div>
+                <div class="col-md-8">
+                    <h4 class="font-weight-bold">{{ pm.fullName }}</h4>
+                    <h5>{{ pm.role }}</h5>
+                    <span>
+                        <i></i>
+                        <a href="{{ pm.website }}" target="_blank">Website</a></span
+                    >
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+<div>
+    <h3 class="mb-3 font-weight-bold">{{ 'aboutUsOverview.getInTouch' | translate }}</h3>
+    <span jhiTranslate="releaseNotes"></span>
+    <ul>
+        <li>
+            <a [href]="bugReportUrl" rel="noreferrer noopener" class="footer-text ml-3" target="_blank">{{ 'bug' | translate }}</a>
+        </li>
+        <li>
+            <a [href]="featureRequestUrl" class="footer-text ml-3" target="_blank"> {{ 'feature' | translate }}</a>
+        </li>
+        <li>
+            <a [href]="email" class="footer-text ml-3" target="_top"> {{ 'contact' | translate }}</a>
+        </li>
+    </ul>
+</div>
+<div class="mt-5" *ngIf="data && data.contributors">
+    <h3 class="mb-3 font-weight-bold">{{ 'aboutUsOverview.contributors' | translate }}</h3>
+    <div class="row">
+        <div class="col-sm-1 grid-margin" *ngFor="let contributor of data.contributors">
```

Comment:
Is this ordered in any particular order? What about a random order on each pageload?

Reply:
The json file was build based on the sorting of github contributors, I think there its ordered based on amount of commits. Of course I can list the contributors randomly if you prefer so